.github/renovate.json5

```diff
@@ -13,13 +13,14 @@
     "ignorePaths": [
     ],
 
-    "ignoreDeps": [
-    ],
+    "ignoreDeps": ["avaplatform/avalanchego],
```

Comment:
this ignoreDeps won't cover "100%" of potential updates for the same rational given below on matchPackagePatterns, so we're relying on the "enabled: false" we'll set below, maybe redundant?